frontend/src/pages/home.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Image from 'next/image';
+import BitcoinLogo from '../assests/svgs/bitcoin-logo.svg';
+import Machines from '../assests/images/machines.webp';
+
+const HomePage = React.memo(function HomePage() {
+    const wrapperStyles: React.CSSProperties = {
```

Comment:
we can put these properties in a separate style file and import them here.

Reply:
Alright, preferably why don't I create a CSS file in that folder and pass the required styles

Reply:
That's okay too